pkg/app/src/components/TabPanels/Organization/tmpOverview.tsx

```diff
@@ -43,6 +45,13 @@ export function TmpOverview() {
 			if (state) {
 				// The transaction was successful, clear state
 				tmpOrgState?.clearAll()
+				result.events.forEach(({ event: { data, method, section } }) => {
+					if (section === 'control' && method === 'OrgCreated') {
+						setTimeout(function () {
+							push(`/organisations/${data[1].toHex()}/dashboard`)
+						}, 3000)
```

Comment:
Shall we do it like that? Could be that after 3 seconds the GraphQL didn't index the organization.
You could create a subscription to the organization id via GraphQL, if it exists, you can redirect the user to the overview page.

Otherwise it's not really reliable.

Reply:
yes you're right I'll update it